config/routes.rb

```diff
@@ -111,7 +111,21 @@
     get "/file-attachments/:file_attachment_id/delete" => "file_attachments#confirm_delete", as: :confirm_delete_file_attachment
   end
 
-  get "/healthcheck", to: "healthcheck#index"
+  get "/healthcheck", to: GovukHealthcheck.rack_response(
+    GovukHealthcheck::SidekiqRedis,
+    GovukHealthcheck::ActiveRecord,
+    Healthcheck::GovernmentDataCheck,
+    Healthcheck::ActiveStorage,
+  )
+
+  get "/healthcheck/live", to: proc { [200, {}, %w[OK]] }
+  get "/healthcheck/ready", to: GovukHealthcheck.rack_response(
+    GovukHealthcheck::ActiveRecord,
+    GovukHealthcheck::SidekiqRedis,
+  )
+
+  get "/healthcheck/active-storage", to: "healthcheck#active_storage"
```

Comment:
I think moving this to a separate endpoint conflicts with the [Continuous Deployment RFC production healthy checks](https://github.com/alphagov/govuk-rfcs/blob/main/rfc-128-continuous-deployment.md#check-app-is-healthy-ie-it-can-run-in-a-production-environment) that includes checking external services such as S3. 

I guess since there is no exception handling set-up for S3/ActiveStorage we should consider that an essential service given any pages that utilise it would 500 if S3 access was lost, similar to losing a database.

Reply:
Ok, I was mistaken in thinking this wasn't a critical dependency then.  I'll open another PR moving it back to `/healthcheck/ready`.

Reply:
- https://github.com/alphagov/govuk-puppet/pull/11060
- https://github.com/alphagov/content-publisher/pull/2304